sogou-search-api.gemspec

```diff
@@ -23,8 +23,9 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'savon', '~> 2.12'
 
-  spec.add_development_dependency 'bundler', '~> 1.14'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_development_dependency 'simplecov', '~> 0.14'
+  spec.add_development_dependency 'bundler', '~> 2.1'
+  spec.add_development_dependency 'rake', '~> 13.0'
```

Comment:
these two are needed, but they jump over major versions. Given this is a public gem, are we sure this will not introduce breaking changes?

Reply:
Changed to be a major release

Reply:
ok, all good from my side then. Let's see what Babak says